garrysmod/lua/includes/modules/team.lua

```diff
@@ -146,6 +152,20 @@ function GetPlayers(index)
 
 end
 
+function GetAlivePlayers(index)
+
+	local TeamAlivePlayers = {}
+
+	for id,pl in ipairs( player.GetAll() ) do

```

Comment:
- `id` is unused here.
- `player.GetAll` can be replaced by [`player.Iterator`](https://wiki.facepunch.com/gmod/player.Iterator).

Reply:
[https://wiki.facepunch.com/gmod/player.Iterator](player.Iterator) is currently only in the Dev branch, so until it's released to public, didn't think it'd be a good idea to make another pull request that's dependent on another item. I could swap out id for _ but has to have something so it can loop through each iterance.

Reply:
> [`player.Iterator`](https://wiki.facepunch.com/gmod/player.Iterator) is currently only in the Dev branch, so until it's released to public, didn't think it'd be a good idea to make another pull request that's dependent on another item.

Since it's "better" (imo) to test your changes with the dev branch and assuming your changes will be rolled out together with this new function, I think you can use it without any concern 👍